authors/apps/authentication/views.py

```diff
@@ -1,16 +1,16 @@
-from rest_framework import status
-from rest_framework.generics import RetrieveUpdateAPIView
+from rest_framework import status, generics
+from rest_framework.generics import RetrieveUpdateAPIView,CreateAPIView
```

Comment:
Hi @FawazFarid. Thanks for the feedback. I have added the suggested space ✅ . I used this [Django REST Swagger documentation](https://django-rest-swagger.readthedocs.io/en/latest/) to write the documentation for the app. I have replaced the `APIView` with `generics` views. This is because [Django REST Swagger ](https://django-rest-swagger.readthedocs.io/en/latest/) is able to auto generate schema and parameters with generic views.